packages/shared/src/consts/severity.ts

```diff
@@ -0,0 +1,6 @@
+export enum SeverityLevel {
```

Comment:
Not critical, but usually we suffix with Enum, 
```suggestion
export enum SeverityLevelEnum {
```